i18n/it/cosmic_settings.ftl

```diff
@@ -241,195 +245,195 @@ x-hours = { $number ->
 ## Desktop: Workspaces
 
 workspaces = Spazi di lavoro
-    .desc = Imposta numero, comportamento e posizione degli spazi di lavoro.
+    .desc = Imposta numero spazi di lavoro, il loro comportamento e la loro posizione.
 
 workspaces-behavior = Comportamento spazi di lavoro
     .dynamic = Spazi di lavoro dinamici
-    .dynamic-desc = Rimuovi automaticamente spazi di lavoro vuoti.
-    .fixed = Numero fisso di spazi di lavoro
-    .fixed-desc = Aggiungi o rimuovi spazi di lavoro nell'anteprima.
+    .dynamic-desc = Rimuovi automaticamente gli spazi di lavoro vuoti
+    .fixed = Spazi di lavoro limitati
+    .fixed-desc = Aggiungi o rimuovi gli spazi di lavoro
 
-workspaces-multi-behavior = Comportamento con più schermi
-    .span = Gli spazi di lavoro comprendono più schermi
-    .separate = Ogni schermo ha il suo spazio di lavoro
+workspaces-multi-behavior = Comportamento multi-schermo
+    .span = Gli spazi di lavoro si espandono in tutti gli schermi
+    .separate = Ogni schermo ha gli spazi di lavoro separati
 
-workspaces-overview-thumbnails = Anteprime Spazio di Lavoro
-    .show-number = Mostra Numero Spazio di Lavoro
-    .show-name = Mostra Nome Spazio di Lavoro
+workspaces-overview-thumbnails = Anteprima spazi di lavoro
+    .show-number = Mostra il numero dello spazio di lavoro
+    .show-name = Mostra il nome dello spazio di lavoro
 
-workspaces-orientation = Orientamento Spazi di Lavoro
+workspaces-orientation = Orientamento spazi di lavoro
     .vertical = Verticale
     .horizontal = Orizzontale
 
 ## Networking: Wired
 
-wired = Cablata
-    .desc = Connessione cablata, profili di connessione
+wired = Via cavo
+    .desc = Connessione via cavo, profili di connessione
 
 ## Networking: Online Accounts
 
-online-accounts = Account Online
-    .desc = Aggiungi account, IMAP e SMTP, login aziendali
+online-accounts = Account online
+    .desc = Aggiungi account IMAP, SMTP e enterprise
 
 ## Time & Language
 
-time = Ora e Lingua
+time = Orario e linguaggio
     .desc = N/A
 
 time-date = Data e ora
-    .desc = Fuso orario, impostazioni orologio automatiche, e alcuni formati ora.
+    .desc = Fuso orario, impostazioni orologio automatico e formattazione orario
     .auto = Imposta automaticamente
 
 time-zone = Fuso orario
     .auto = Fuso orario automatico
-    .auto-info = Richiede geolocalizzazione e accesso a internet
+    .auto-info = Richiede i servizi di posizione e l'accesso a internet
 
 time-format = Formato data e ora
-    .twenty-four = formato 24 ore
+    .twenty-four = Formato 24 ore
     .first = Primo giorno della settimana
 
-time-region = Regione e Lingua
-    .desc = Formato date, ora e numeri in base alla tua regione
+time-region = Lingua e località
+    .desc = Formattazione data, ora e numeri basata sulla località
 
 ## Sound
 
 sound = Suono
     .desc = N/A
 
-sound-output = Uscita
-    .volume = Volume in uscita
-    .device = Dispositivo in uscita
-    .level = Livello in uscita
+sound-output = Output
```

Comment:
In elementary school (consider that I'm 23) the IT teacher would teach us about "input" and "output".
Its term is constantly being used in every device that exists.

I mean, why would we have more accessible translations that are less precise, while english (original) translations are "more tech-savvy"?
It's ultimately a skill-issue of out Country, we should fix that, not embrace it.

I use english interfaces because they're 100% precise (as intended), otherwise I'd use italian interfaces.
I'm not the only one that despise the badly-translated italian versions of a lot of programs, at some point setting everything to english might be a bit alienating because we're italians and we'd like a good usage of our language, since it's more complete than english.

Reply:
I wouldn't say less precise though. In computer engineering classes and books (not long ago, I'm 31) everything was translated in Italian. Even "computer" which is so common was translated into "calcolatore". So in practice both versions are correct. But I agree that's common to leave it in English in some translations, so it's more a matter of preference. I'm fine with both in this case, while preferring the Italian version.

And yeah I also use English in all my devices, translations always sound wrong.